Doc/library/subprocess.rst

```diff
@@ -76,6 +76,14 @@ compatibility with older versions, see the :ref:`call-function-trio` section.
    The *universal_newlines* argument is equivalent  to *text* and is provided
    for backwards compatibility. By default, file objects are opened in binary mode.
 
+   If the current process recevies a :exc:`KeyboardInterrupt` and
+   *cleanup_timeout* is non-zero, the child process is given additional
+   time to finish before it is killed, to allow potential clean-up operations
+   in the child to complete. During this time, a second :exc:`KeyboardInterrupt`
+   will kill the child immediately. If *cleanup_timeout* is not specified,
```

Comment:
Current implementation looks like it will leave the child running, rather than killing it. Opportunity for another test case :)

Reply:
Ouch, that's embarrassing.  Thanks for catching it.  I've pushed a new commit which fixes that... with a test case. ;-)